scripts/istio-images-mirror

```diff
@@ -30,3 +30,4 @@ quay.io/calico/cni rancher/calico-cni v3.10.2
 quay.io/calico/kube-controllers rancher/calico-kube-controllers v3.10.2
 quay.io/calico/pod2daemon-flexvol rancher/calico-pod2daemon-flexvol v3.10.2
 us.gcr.io/k8s-artifacts-prod/external-dns/external-dns rancher/kubernetes-external-dns v0.6.0
+quay.io/open-policy-agent/gatekeeper rancher/opa-gatekeeper v3.1.0-beta.7
```

Comment:
@prachidamle Do not remove the line feed as this will break the mirror script

Reply:
@StrongMonkey updated